### PR TITLE
Add Support for BLIS+libFLAME toolchain

### PR DESCRIPTION
**Pull request by SebastianAchilles**
This pull request adds support for a toolchain using GNU with OpenMPI, BLIS, libFLAME, ScaLAPACK and FFTW.

I am not sure if the name `gobff` is the best. I tried to be in line with the other naming of toolchains, like `goblf`.

### CODE REVIEW

File: easybuild/toolchains/linalg/blis.py [hunk 1]
```diff
@@ -42,3 +44,15 @@ class Blis(LinAlg):
...
     BLAS_FAMILY = TC_CONSTANT_BLIS
+
+    def _set_blas_variables(self):
+        """AMD's fork with version number > 2.1 names the MT library blis-mt, while vanilla BLIS doesn't."""
+
+        # This assumes that AMD's BLIS has ver > 2.1 and vanilla BLIS < 2.1
+
+        found_version = self.get_software_version(self.BLAS_MODULE_NAME)[0]
+        ver = LooseVersion(found_version)
+        if ver > LooseVersion('2.1'):
```
Comment (akesandgren) [inline easybuild/toolchains/linalg/blis.py:55]: no need to create a variable if it's only used once, so reduce to
if LooseVersion(found_version) > LooseVersion('2.1'):

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: easybuild/toolchains/linalg/blis.py (hunk 1 above)
Comment (SebastianAchilles) [reply easybuild/toolchains/linalg/blis.py]: Thanks for the comment! I changed it.

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @SebastianAchilles!

**Comment by akesandgren** [comment]
Now we just need some easyconfigs to go with it... or are they already there?

**Comment by SebastianAchilles** [comment]
Yes, I create a PR for the easyconfigs:
https://github.com/easybuilders/easybuild-easyconfigs/pull/11761

**Pull request merged**